Subject: link libiscsi with -lpthread for iser
From: Michael Tokarev <mjt@tls.msk.ru>
Date: Mon, 31 Dec 2018 21:42:57 +0300

When built with iser support, libiscsi uses symbols from libpthread.
So link in -lpthread too.

--- a/lib/Makefile.am	2016-10-09 21:54:23.000000000 +0300
+++ b/lib/Makefile.am	2018-12-31 21:40:46.165146563 +0300
@@ -25 +25 @@
-libiscsi_la_LDFLAGS += -libverbs -lrdmacm
+libiscsi_la_LDFLAGS += -libverbs -lrdmacm -lpthread
